src/renderers/dom/shared/hooks/ReactDOMUnknownPropertyHook.js

```diff
@@ -35,39 +35,21 @@ function getStackAddendum(debugID) {
 }
 
 if (__DEV__) {
-  var reactProps = {
-    children: true,
-    dangerouslySetInnerHTML: true,
-    key: true,
-    ref: true,
-
-    autoFocus: true,
-    defaultValue: true,
-    defaultChecked: true,
-    innerHTML: true,
-    suppressContentEditableWarning: true,
-    onFocusIn: true,
-    onFocusOut: true,
-  };
   var warnedProperties = {};
   var EVENT_NAME_REGEX = /^on[A-Z]/;
+  var ARIA_NAME_REGEX = /^aria-/i;
```

Comment:
This is new. Since we allow any attribute, and now care about casing, the unknown property hook was picking up on incorrectly cased aria attributes, so we need to skip them. It kind of makes me wonder if we should just go ahead and remove the aria hook and do the ARIA checks in here.